src/homepage/homepage.html

```diff
@@ -45,14 +95,19 @@
     }
 
     #activities div:nth-child(2n + 1) {
-      background-color: var(--tertiary-color);
+      background-color: var(--primary-color);
     }
 
     #merchandise img {
       width: 450px;
       max-width: calc(100% - 20px);
     }
   </style>
+  <section id="logo">
+    <img src="../../images-optimized/logo.svg" alt="#Trending logo">
```

Comment:
width and height are sort of irrelevant since css uses `vw` to determine the size, so a reflow will happen anyways